### ISSUE TITLE
site.url is missing in the generated search index

### ISSUE THREAD

**Issue by pavelanni** [Mogztter/antora-lunr#61]
The search functionality stopped working for me since 0.7.1. It looks like the `site.url` which I set in the `antora-playbook.yml` isn't passed to the search module. Here is my test: Antora Demo Site. I built the site with `DOCSEARCH_ENABLED=true DOCSEARCH_ENGINE=lunr DOCSEARCH_INDEX_VERSION=latest NODE_PATH="$(npm -g root)" antora --generator antora-site-generator-lunr antora-playbook.yml`. When I search for a word and click on one of the variants it gives me a URL like this: `file:///component-b/2.0/module-one/overview.html` -- i.e. without the `site.url` part which I set in the playbook to `file:///Users/pavela/tmp/docs-site/build/site`.

I'm running the latest versions of everything:
npm ls -g --depth 0
/Users/pavela/.nvm/versions/node/v12.17.0/lib
├── @antora/cli@2.3.3
├── @antora/site-generator-default@2.3.3
├── antora-lunr@0.7.1
├── antora-site-generator-lunr@0.5.0
└── npm@6.14.4

Last time it worked properly was about a month ago (I didn't test since then). My suspicion is that it might be caused by this commit: https://github.com/Mogztter/antora-lunr/commit/364a957b5abd8892841a08bda7aca8b376d2560e   

Appreciate your help.

**Comment by Mogztter** [comment]
Please read https://github.com/Mogztter/antora-lunr/pull/59 and https://github.com/Mogztter/antora-lunr/releases/tag/v0.7.0

tl;dr; you need to update the UI.

**Comment by pavelanni** [comment]
That helped! Thank you, Guillaume!

**Comment by Mogztter** [comment]
Awesome! :+1:

**Comment by pavelanni** [comment]
Hi Guillaume, just an idea to think about: it would be very helpful to be able to preserve customization I did in the `supplemental_ui`. In my case, I've added a logo instead of the title and removed the "Services" menu. But when I updated it, my changes were gone. Maybe I should've looked at the diff first and added only the changes instead of blindly updating the whole `supplemental_ui`. Or maybe there is a better way. Thanks!

**Comment by Mogztter** [comment]
Hi Pavel, as far as I know this is currently the "best workaround". Alternatively, we could describe each changes to be done in the README but that would be tedious.
Another idea, would be to provide git diff/patchs but again not ideal if the patch produces conflicts.

I believe that the next major release of Antora will introduce an extension mechanism and maybe a better way to extend the base UI.
